py-polars/Cargo.toml

```diff
@@ -104,6 +104,7 @@ all = [
   "parquet",
   "ipc",
   "polars-python/all",
+  "performant",
```

Comment:
Code-path for left join on sorted keys is also affected, but it is gated behind the `performant` feature flag so it doesn't get compiled in debug builds.

I think maybe instead of adding it here, we can add a test step to the Python release workflow that runs through the test suite with the optimized release build?

Reply:
I assumed it was always included in pypolars. It should always be activated.